client/state/user-settings/README.md

```diff
@@ -0,0 +1,23 @@
+User Settings
+=============
+
+This store holds user settings from `/me/settings`.
+
+# Usage
+
+1. Render `QueryUserSettings` from `components/data/query-user-settings`
+2. Connect your component specifying proper setting names:
+```es6
```

Comment:
A few months back I updated a few (maybe a lot) of the readmes to `es6` syntax where they previously had no highlight tag - At the time some ES6+ specific syntax wasn't supported with the JS tag.
I'm pretty sure that at some point either Github or PrismJS (can't remember which) stopped highlighting `ES6` tags but at the same time `JS` was updated to cover the `es6` syntax.
I've just checked and [devdocs highlights es6](https://wpcalypso.wordpress.com/devdocs/client/lib/devices/README.md?term=es6)  properly, as does Github - but I think I had to alias `es6` to `js` for the devdocs syntax highlighting to work.
My memory is fuzzy on this though - it was a little while ago now!

`es6` works but I agree it should be `js` mostly just for future-proofing.